### ISSUE TITLE
[Accessibility] [DetailsList] Validation fails for check column header button name

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1078]
The details list component fails button name aria validation for the column header for check box.

Details:
https://dequeuniversity.com/rules/axe/2.1/button-name

Error displayed:
![image](https://cloud.githubusercontent.com/assets/11937034/23212671/c8d208de-f92d-11e6-9d77-35cc8de626b3.png)

**Comment by mdahamiwal** [comment]
setting `ariaLabelForSelectAllCheckbox` option in details list should fix this.

**Comment by prmittal** [comment]
Thanks. I missed the setting.
